fedimint-api/src/encoding/btc.rs

```diff
@@ -37,6 +38,18 @@ impl_encode_decode_bridge!(bitcoin::Txid);
 impl_encode_decode_bridge!(bitcoin::util::merkleblock::PartialMerkleTree);
 impl_encode_decode_bridge!(bitcoin::util::psbt::PartiallySignedTransaction);
 
+impl<T: MiniscriptKey> Encodable for Descriptor<T> {
+    fn consensus_encode<W: Write>(&self, writer: &mut W) -> Result<usize, Error> {
+        SerdeEncodable(self.clone()).consensus_encode(writer)
```

Comment:
How stable and future-proof is the serde representation of miniscrit's Descriptor? Could we use maybe something more stable?

Possibly maybe we should implement `Encodable` for this ourselves, explicitly handling each variant we support so we can make sure constants don't change?

I don't know ... just thinking aloud. In 6 months miniscript developers might tweak the representation as they don't expect it to be *consensus* stable, and we will be kind of stuck.

Reply:
I should be able to do this since we only use one kind of multisig descriptor.

Reply:
We might want to put it in an enum just to leave a room for future extension.

Reply:
The next change will probably be to switch to taproot then we can get rid of this implementation.